client/lobby/SelectionTab.cpp

```diff
@@ -421,6 +498,29 @@ void SelectionTab::select(int position)
 	else if(position >= listItems.size())
 		slider->scrollBy(position - (int)listItems.size() + 1);
 
+	if(curItems[py]->isFolder) {
+		if(curItems[py]->folderName == "..")
+		{
+			std::vector<std::string> filetree;
+			boost::split(filetree, curFolder, boost::is_any_of("/"));
+			filetree.erase(filetree.end());
+			filetree.erase(filetree.end());
```

Comment:
end() points at element after last, I think this is a reason of crash. If you want to remove last element from vector, I propose to use pop_back() method.

Reply:
I've changed it. Can you do a retest?

I cannot reproduce this crash. Tried it with Ubuntu and Windows. Also tried some special Chars.

Maybe MacOS specific?

Reply:
Update: Was able to reproduce it on android. Fix remove issue. Should work now. :)